src/Pages/HomePage/PageSections/BrandsSection.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import '../HomePage.css'
+import BrandImage from '../../../Components/BrandImage/BrandImage'
+import Tencent from "../../../Assets/Brands/tencent.png"
+import Sony from "../../../Assets/Brands/sony.png"
+import Nvidea from "../../../Assets/Brands/nvidea.png"
+import Intel from "../../../Assets/Brands/intel.png"
+import EA from "../../../Assets/Brands/electronics-arts.png"
+
+const BrandsSection = () => {
+    const featuredBrands = [{
+        imgSrc: Nvidea,
+        altTxt: "nvidea brand logo"
+    }, {
+        imgSrc: Intel,
+        altTxt: 'intel brand logo'
+    }, {
+        imgSrc: EA,
+        altTxt: 'Electronice Arts Games logo'
+    }, {
+        imgSrc: Tencent,
+        altTxt: 'Tencent brand logo'
+    }, {
+        imgSrc: Sony,
+        altTxt: 'Sony Brand logo'
+    },
+    ]
+    return (
+        <section className="section deals-container mg-t-70">
+            <h1 className="new-font center-txt">
+                FEATURED BRANDS
```

Comment:
Suggestion: As far as I know having only one `h1` tag on your page considered good practice but you have used in your all homepage sections, do let me know if I am wrong

Reply:
not sure if its the best practice but apparently it helps with seo so as long as it makes sense for a h1 you can use it there